tests/e2e.sh

```diff
@@ -25,7 +25,7 @@ esac
 
 KUBERNETES_VERSION=v1.18.6
 KUBE_STATE_METRICS_LOG_DIR=./log
-KUBE_STATE_METRICS_IMAGE_NAME="quay.io/coreos/kube-state-metrics-${ARCH}"
+KUBE_STATE_METRICS_IMAGE_NAME="k8s.gcr.io/kube-state-metrics/kube-state-metrics-${ARCH}"
```

Comment:
small nitpick: k8s.gcr.io supports multi-arch with 1.9.8 so we could omit the -${ARCH} here

Reply:
I am thinking of going back to quay as the default image